package-scripts/datadog-agent/prerm

```diff
@@ -28,8 +28,11 @@ else
     exit 1;
 fi
 
-# Delete all.pyc files
+# Delete all.pyc files in the agent's dir
 find /opt/datadog-agent/agent -name '*.py[co]' -type f -delete || echo 'Unable to delete .pyc files'
 
-exit 0
+# Delete all the .pyc files in the embedded dir that are part of the agent's package
+# (i.e. not from packages installed manually by the user)
+cat /opt/datadog-agent/embedded/py_compiled_files.txt | xargs rm -f
```

Comment:
@truthbk We wouldn't delete anything in that case.

For a safer approach we could add a `grep /opt/datadog-agent/embedded/` filter on the contents of the file to make sure that we don't delete anything outside of the embedded dir.

But in general we're quite vulnerable to any change on the `py_compiled_files.txt` file.